modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyEntriesMVCActionCommand.java

```diff
@@ -39,41 +43,49 @@
 import org.osgi.service.component.annotations.Reference;
 
 /**
- * @author Marco Galluzzi
+ * @author Sam Ziemer
  */
 @Component(
 	property = {
 		"javax.portlet.name=" + DLPortletKeys.DOCUMENT_LIBRARY,
 		"javax.portlet.name=" + DLPortletKeys.DOCUMENT_LIBRARY_ADMIN,
 		"javax.portlet.name=" + DLPortletKeys.MEDIA_GALLERY_DISPLAY,
-		"mvc.command.name=/document_library/copy_folder"
+		"mvc.command.name=/document_library/copy_entries"
 	},
 	service = MVCActionCommand.class
 )
-public class CopyFolderMVCActionCommand extends BaseMVCActionCommand {
+public class CopyEntriesMVCActionCommand extends BaseMVCActionCommand {
```

Comment:
coping a mix of docs and folder does not work:

first the "copy from" is empty and it gives an ok but the files/folders does not appear on the "copy to" folder


https://github.com/liferay-lima/liferay-portal/assets/47524751/0b83475a-a6dc-42d4-a68f-a921929d425a

Reply:
Works fine for me. There was an error log that had no effect (I'll add an additional commit fixing that).

About the lack of text in "Copy From", I need to speak with design, as that field doesn't make much sense when copying multiple elements. We can ignore it for the moment.